src/models/Member.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const memberSchema = new Schema({
+  firstName: {
+    type: String,
+    require: true,
+  },
+  lastName: {
+    type: String,
+    require: true,
+  },
+  dni: {
+    type: Number,
```

Comment:
it should be string, fix joi validations too

Reply:
type change to string